javascript/new-src/.jshintrc

```diff
@@ -5,9 +5,9 @@
   "latedef": true,
   "unused": true,
   "noarg": true,
-  "indent": 1,
+  "indent": 2,
   "trailing": true,
-  "white": true,
```

Comment:
are you aware what [white](https://github.com/jshint/jshint.github.io/blob/1d33ae6996be304d413164b796ccd9671415ec70/options/legacy.table) does?